### PR TITLE
Updated mod abuse filter and !q all looks

### PR DESCRIPTION
**Pull request by Koramund**

### CODE REVIEW

**Comment by Koramund** [comment]
I hate github and I hate you for using rebases

File: src/packages/quote.py [hunk 1]
```diff
@@ -2,10 +2,13 @@
 
 # Setup python logging
 import logging
+
+from main import start
```
Comment (juliavdkris) [inline src/packages/quote.py]: Quoi?

(Isn't used anywhere as far as I can see)

**Comment by juliavdkris** [comment]
> I hate github and I hate you for using rebases

<3

File: src/packages/quote.py (hunk 1 above)
Comment (Koramund) [reply src/packages/quote.py]: 🤔

Comment (Koramund) [reply src/packages/quote.py]: Deleted some unused imports, maybe I'll do an actual clean-up later? who knows.

**Review by juliavdkris — APPROVED** [review]

**Pull request merged**